lib/web_ui/dev/build.dart

```diff
@@ -20,6 +20,11 @@ class BuildCommand extends Command<bool> {
         abbr: 'w',
         help: 'Run the build in watch mode so it rebuilds whenever a change'
             'is made.',
+      )
+      ..addOption(
+        'goma',
+        abbr: 'j',
+        help: 'Enable parallelization through goma',
```

Comment:
Are you sure this is a goma option and not ninja in general? I thought even without goma this option would speed up your ninja builds by parallelizing across multiple CPU cores.

Reply:
Thanks for pointing this out. You are right, this is a ninja parallelization option that it inherited from Make. I'll update the text and variable names accordingly.